www/views/confirm.html

```diff
@@ -77,18 +77,21 @@
           </div>
           <i class="icon bp-arrow-right"></i>
         </a>
-        <a class="item single-line item-icon-right" ng-if="!insufficientFunds && !noMatchingWallet" ng-click="showDescriptionPopup()">
+        <div class="item item-icon-right" ng-if="!insufficientFunds && !noMatchingWallet" ng-click="chooseFeeLevel()">
+          <span class="label">{{'Fee' | translate}}: {{feeLevel | translate}}</span>
```

Comment:
Can you please include the ":" in the translatable string?
`{{'Fee:' | translate}}`
Punctuation is not attached to words in French, except for point and comma.